Modules/Sources/BuildSettingsKit/BuildSettings+Live.swift

```diff
@@ -13,6 +13,14 @@ extension BuildSettings {
         explatPlatform = bundle.infoValue(forKey: "WPExplatPlatform")
         itunesAppID = bundle.infoValue(forKey: "WPItunesAppID")
         appURLScheme = bundle.infoValue(forKey: "WPAppURLScheme")
+        about = ProductAboutDetails(
+            twitterHandle: bundle.infoValue(forKey: "WPProductTwitterHandle"),
+            twitterURL: URL(string: bundle.infoValue(forKey: "WPProductTwitterURL"))!,
+            blogURL: URL(string: bundle.infoValue(forKey: "WPProductBlogURL"))!
+        )
+        zendeskSourcePlatform = bundle.infoValue(forKey: "WPZendeskSourcePlatform")
+        mobileAnnounceAppID = bundle.infoValue(forKey: "WPMobileAnnounceAppID")
+        authKeychainServiceName = bundle.infoValue(forKey: "WPAuthKeychainServiceName")
```

Comment:
All of these are just hard-coded values which don't have anything to do with build configurations. Maybe we can just have some readonly variables or something? I don't think all these values need to be in the Info.plist files.

Reply:
> All of these are just hard-coded values which don't have anything to do with build configurations

True. However, they do change from WordPress and Jetpack. Maybe we could derive them from `brand`/`WPAppBrand`. But then we'd have sort of two places where to place the values: `Info.plist` and the code `switch brand { case ...` here.

Reply:
>  two places where to place the values: Info.plist and the code switch brand { case ... here.

💯. The idea is to specify configuration in data: `Info.plist` and `.xcconfig`. It's not always feasible, but you generally shouldn't have to search the codebase to find these values – it should be in configuration files, in data. I used `switch brand` sparingly.

There are multiple advantages to storing these values in data and not code: easier to find, can be modified without recompiling the app, easier to edit with scripts if needed.